plugins/scaffolder-common/api-report.md

```diff
@@ -61,7 +62,37 @@ export interface TaskStep {
   name: string;
 }
 
-// @public (undocumented)
+// @public @deprecated
+export interface TemplateEntityV1beta2 extends Entity {
+  // (undocumented)
+  apiVersion: 'backstage.io/v1beta2';
+  // (undocumented)
+  kind: 'Template';
+  // (undocumented)
+  spec: {
+    type: string;
+    parameters?: JsonObject | JsonObject[];
+    steps: Array<{
+      id?: string;
+      name?: string;
+      action: string;
+      input?: JsonObject;
+      if?: string | boolean;
+    }>;
+    output?: {
+      [name: string]: string;
+    };
+    owner?: string;
+  };
+}
+
+// @public @deprecated
+export const templateEntityV1beta2Schema: JSONSchema;
```

Comment:
Should not be exporting this one right? and we're getting rid of `templateEntityV1beta3Schema`

Reply:
Oh did we decide on removing both? Yeah sure then I can do a breaking insta-delete instead of a deprecation if you prefer that?